syntax = "proto3";

message Ack {}

message ToComputer {
  oneof message {
    Ack ack = 1;
  }
}